### PR TITLE
Chinese_Traditional.xml: fully reviewed and localized

### PR DESCRIPTION
**Pull request by pan93412**
## README

Due to the not-good translation quality in the current version, I reviewed all the strings and improved the translation. By the way, I really loved this software! It helped me cool down my Mac a lot. Thanks @crystalidea! :)

## Screenshot

截圖 2021-05-24 下午6 00 12
截圖 2021-05-24 下午6 00 19
截圖 2021-05-24 下午6 00 28
截圖 2021-05-24 下午6 00 39
截圖 2021-05-24 下午6 00 47

### CODE REVIEW

**Comment by kleuter** [comment]
Hello and thanks a lot!

You changed basically everything. Are you sure it was that bad? UpdateChecker/LetsMoveMac/QLineEditEx as well?

Regards.

File: languages/Chinese_Traditional.xml [hunk 1]
```diff
@@ -1,195 +1,196 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-<LanguageStrings version="1.5.5" Author="Tenjin">
+<?xml version="1.0" encoding="UTF-8"?>
+<LanguageStrings version="1.5.5" Author="Yi-Jyun Pan, Tenjin">
```
Comment (kleuter) [inline languages/Chinese_Traditional.xml:2]: Should be 1.5.9

**Comment by pan93412** [comment]
> Hello and thanks a lot!
> 
> You changed basically everything. Are you sure it was that bad? UpdateChecker/LetsMoveMac/QLineEditEx as well?
> 
> Regards.

I prefer to read it again in my mind and check if it is not fluent. Some of them are just like from Chinese (Simplified), which we don't prefer to translate to.

**Comment by pan93412** [comment]
@kleuter could you check it again? thanks!

**Comment by pan93412** [comment]
> > Hello and thanks a lot!
> > You changed basically everything. Are you sure it was that bad? UpdateChecker/LetsMoveMac/QLineEditEx as well?
> > Regards.
> 
> I prefer to read it again in my mind and check if it is not fluent. Some of them are just like from Chinese (Simplified), which we don't prefer to translate to.

Oh. I also compared the translation with English's original one. I found there are some translations which seems like not matched the source string. I fixed up them too.

**Comment by kleuter** [comment]
Thank you.
Please contact me via email at support for a license.

**Comment by kleuter** [comment]
the file though fails to be a valid XML unfortunately

**Comment by kleuter** [comment]
image

**Comment by pan93412** [comment]
> the file though fails to be a valid XML unfortunately

well... I've tested by copying this XML to my program instance.

**Comment by kleuter** [comment]
image

**Comment by kleuter** [comment]
This must be deleted I suppose.

File: languages/Chinese_Traditional.xml [hunk 2]
```diff
@@ -1,195 +1,196 @@
...
+        <buttonFanControlCustom>客製化⋯</buttonFanControlCustom>
+        <buttonFanControlCustomInfo>客製化控制模式</buttonFanControlCustomInfo>
         <buttonMinimize>最小化</buttonMinimize>
-        <buttonMinimizeMac>隱藏到選單列</buttonMinimizeMac>
-        <buttonPreferences>設定...</buttonPreferences>
-        <buttonAbout>關於...</buttonAbout>
-        <messageHelperToolInstall>Macs Fan Control 需要安裝一個幫助程序來控制風扇. 如果你只是只想監視風扇溫度傳感器則可以跳過這個安裝.</messageHelperToolInstall>
-        <messageHelperToolUpgrade>Macs Fan Control需要安裝一個更新的版本的幫助程式來控制風扇.</messageHelperToolUpgrade>
-        <messageFanControlNotAvailableTitle>風扇控制不可用</messageFanControlNotAvailableTitle>
-        <messageFanControlNotAvailable>Macs Fan Control 無法連接至輔助程序以便開啟風扇控制並且調整其運轉速度。\n\n請確認每當要求的時候，務必要輸入管理員的密碼。\n\nb你仍然可以在監視模式下使用此應用程式.</messageFanControlNotAvailable>
```
Comment (pan93412) [inline languages/Chinese_Traditional.xml:27]: try to add the character "b" from "\nb" back? I'm not sure whether it is an additional or required character.

**Comment by pan93412** [comment]
> This must be deleted I suppose.

Could you help me remove this? I didn't notice it when I editing it in my VS Code. Sorry :(

**Comment by kleuter** [comment]
No problem, I removed 0x08, it's an invalid ASCII character.

**Comment by kleuter** [comment]
https://github.com/crystalidea/macs-fan-control/commit/9abbe9c86b57084f4c91c07832339d28385991af

**Pull request merged**